src/app/clusters/webrtc-transport-requestor-server/webrtc-transport-requestor-server.h

```diff
@@ -29,14 +29,17 @@
 #include <string>
 #include <vector>
 
+using ICEServerDecodableStruct = chip::app::Clusters::Globals::Structs::ICEServerStruct::DecodableType;
+using WebRTCSessionStruct      = chip::app::Clusters::Globals::Structs::WebRTCSessionStruct::Type;
+using ICECandidateStruct       = chip::app::Clusters::Globals::Structs::ICECandidateStruct::Type;
+using StreamUsageEnum          = chip::app::Clusters::Globals::StreamUsageEnum;
+using WebRTCEndReasonEnum      = chip::app::Clusters::Globals::WebRTCEndReasonEnum;
```

Comment:
No global using, please.

Reply:
Moved inside the WebRTCTransportRequestor namespace.